Doc/library/asyncio-task.rst

```diff
@@ -170,7 +170,7 @@ other coroutines::
    In this documentation the term "coroutine" can be used for
    two closely related concepts:
 
-   * a *coroutine function*: an :keyword:`async def` function;
+   * a *coroutine function*: a :keyword:`async def` function;
```

Comment:
The rule is, if the next word starts with a vowel sound, use "an" ("an elephant"), otherwise if it's a consonant sound, use "a" ("a fox").


This should remain "an" because the next displayed word is "async": "an async def function"
```suggestion
   * a *coroutine function*: an :keyword:`async def` function;
```